src/styles/blocks/card.scss

```diff
@@ -0,0 +1,104 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  flex-grow: 0;
```

Comment:
You don't need flex properties for the card
All elements will be displayed on the new line by default